spec/services/reporting/account_reuse_report_spec.rb

```diff
@@ -148,7 +222,7 @@
           user: create(:user, :fully_registered, registered_at: in_query),
         )
       end
-      (1..3).each do |_|
+      (1..5).each do |_|
```

Comment:
if we're not using the arg we can drop it entirely
```suggestion
      (1..5).each do
```

Reply:
Would this be a good use-case for... ?

```suggestion
      5.times do
```